AGENTS.md

```diff
@@ -38,10 +42,14 @@
   tests should use the pytest.parametrize decorator to enumerate tests cases, specifying
   the appropriate success or failure or exception to be raised for each test as
   appropriate.
-- Tests must be run inside the `pudl-dev` conda environment.
-- For example, the unit tests can be run with `mamba run -n pudl-dev pytest test/unit`.
+- Tests must be run inside the pixi environment.
+- When individual tests are run, we should turn off coverage collection, since otherwise
```

Comment:
nit: usually I just ignore the coverage failures when I run individual tests in dev :shrug: